src/index.ts

```diff
@@ -214,6 +214,13 @@ const createWindow = () => {
         DEFAULT_APP_SETTINGS.accentColor,
       ) as string);
 
+  const linuxMainWindowConf = {
+    icon: asarPath(join(
+      isDevMode ? `${__dirname}../src/` : __dirname,
+      'assets/images/icons/256x256.png',
+    ))
+  };
```

Comment:
Unsure whether we have to exclude this config for non-linux platforms, but I don't have access to others and can't be sure whether it might have adverse effects on windows/mac.

Reply:
Seems fine like this for me, affecting only linux assets.